documentation/modules/ROOT/pages/transformations/event-flattening.adoc

```diff
@@ -303,33 +303,22 @@ Optionally, you can override the field name via `<field name>:<new field name>`,
 When the SMT adds metadata fields to the simplified record's header, it prefixes each metadata field name with a double underscore. For a struct specification, the SMT also inserts an underscore between the struct name and the field name. +
  +
 If you specify a field that is not in the change event record, the SMT does not add the field to the header.
-|===
-
-The following table describes the options that you can specify to configure the event flattening SMT to integrate with the event changes SMT.
-
-
-.Descriptions of event flattening SMT configuration options for integration with event changes SMT
-[cols="30%a,25%a,45%a",subs="+attributes",options="header"]
-|===
-|Option
-|Default
-|Description
 
-|[[extract-new-record-state-header-changed-name]]xref:{link-event-flattening}#extract-new-record-state-header-unchanged-name[`header.unchanged.name`]
+[id="extract-new-record-state-drop-fields-header-name"]
+|xref:{link-event-flattening}#extract-new-record-state-drop-fields-header-name[`drop.fields.header.name`]
 |
-|The Kafka message header to source unchanged field names from.
+|The Kafka message header name to source fields that should be dropped.
```

Comment:
```suggestion
|The Kafka message header name to use for listing field names in the source message that you want to drop from the output message.
```
Does this specify the name of  the header that you want to use to list the fields in the source message that you want to drop in the output message? Or does it represent the name(s) of the headers in the source that you want to drop from the output message?

Reply:
The fields, so I believe your suggestion is correct.